config/locales/en.yml

```diff
@@ -186,6 +186,7 @@ en:
         first_name: *first_name
         last_name: *last_name
         email_confirmation: *email_confirmation
+        disabled_and_deleted_message: 'Invalid Email or password.'
```

Comment:
Nit: content design issue, consider 'Invalid email or password'

Reply:
The message that the devise gem gives, which will appear in the `active? && enabled?` case, has a capital 'E' so for consistency (for the moment) it will need to stay like this.